packages/analytics/analytics-config-store/src/stores/analytics-config.ts

```diff
@@ -10,10 +10,10 @@ export type ConfigStoreState = null | AnalyticsConfigV2
 export const useAnalyticsConfigStore = defineStore('analytics-config', () => {
   let fetchedConfig = false
   const configResult = ref<ConfigStoreState>(null)
```

Comment:
This variable already acts as the config store's loading state.  If it's `null`, the config is still loading.

I think you're concerned about the "no analytics" case.  If the org doesn't have analytics, the config store contains `{ analytics: null, requests: null }` -- in other words, it's possible to distinguish the two cases.